Major code formate to try and set a standard.

Llewellynvdm opened the pull request:
Pull Request for Issue gh-198 .

### Summary of Changes
Made a huge code format to try and insure that we are following the Joomla Standards

### Testing Instructions
Check the standard

### Expected result
Better formating

### Actual result
Was not formated correctly

### Documentation Changes Required
None

ro-ot commented:
@Llewellynvdm this is crazy I can not verify any of this, just to much. I will have to trust you know what you are doing.

Llewellynvdm commented:
Sorry about that, I updated the format to be more like Joomla's standard. I have not change any code, just added four fields as promised to allow JavaScript to be added to a file from with in site and custom admin views.

I will also still add the conditions implementation before pushing out the next update.
